overridable/overridable.go

```diff
@@ -27,23 +28,35 @@ func simpleRule(v interface{}) *rule {
 type complex []map[string]interface{}
 
 type rule struct {
-	pattern  string
+	// pattern is the glob-syntax pattern, such as "a/b/ceee-*"
+	pattern string
+	// patternSuffix is an optional suffix that can be appended to the pattern with "@"
+	patternSuffix string
+
 	compiled glob.Glob
 	value    interface{}
 }
 
 // newRule builds a new rule instance, ensuring that the glob pattern
 // is compiled.
 func newRule(pattern string, value interface{}) (*rule, error) {
+	var suffix string
+	split := strings.Split(pattern, "@")
```

Comment:
is it a bit strange that the `"@"` is hard-coded here although it's a pretty general solution (also in terms of naming)? Not sure, not a blocker anyways

Reply:
Yeah, I see that, but while the solution looks (and _is_) general, it's only used in a single place for a single field so far. And from that angle the effort to make the `@` configurable with the magic `UnmarshalJSON`/`UnmarshalYAML` methods doesn't seem worth it - at the moment. If someone has strong opinions on this, though, I can be convinced.

Reply:
I wouldn't block on it 🤷‍♂️ it's not terrible

Reply:
I think it's fine. If we're ever in a position where we have to figure out how to escape this, other things have probably gone wrong first.

I would be inclined to limit the number of splits, though:

```suggestion
	split := strings.SplitN(pattern, "@", 2)
```

Right now we'd be in danger of throwing away anything after a second (or subsequent) `@`.

Reply:
done!